$OpenBSD: patch-configure,v 1.3 2013/12/19 18:13:48 shadchin Exp $

Remove $latexdir here, so it's possible to set it with kpsewhich(1), as intended.

--- configure.orig	Mon May 20 14:14:35 2013
+++ configure	Fri Jun  7 23:06:56 2013
@@ -3992,7 +3992,6 @@ $as_echo "$ac_res" >&6; }
 if eval test \"x\$"$as_ac_File"\" = x"yes"; then :
 
 else
-  latexdir=/usr/share/texmf/tex/latex
   as_ac_File=`$as_echo "ac_cv_file_$latexdir/base/latex.ltx" | $as_tr_sh`
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $latexdir/base/latex.ltx" >&5
 $as_echo_n "checking for $latexdir/base/latex.ltx... " >&6; }
